kuma/api/v1/tests/test_views.py

```diff
@@ -288,3 +289,48 @@ def test_whoami(user_client, api_settings, wiki_user, beta_testers_group,
         }
     }
     assert_no_cache_header(response)
+
+
+@pytest.mark.django_db
+def test_search_validation_problems(user_client, api_settings):
+    url = reverse('api.v1.search', args=['en-US'])
+
+    # 'q' not present
+    response = user_client.get(url, HTTP_HOST=api_settings.BETA_HOST)
+    assert response.status_code == 400
+    assert response.json()['error'] == "Search term 'q' must be set"
+
+    # 'q' present but falsy
+    response = user_client.get(url, {'q': ''}, HTTP_HOST=api_settings.BETA_HOST)
+    assert response.status_code == 400
+    assert response.json()['error'] == "Search term 'q' must be set"
+
+    # 'q' present but locale invalid
```

Comment:
This is imo a separate test case

Reply:
I'm just lumping 3 checks into 1 python test function. I don't think is particularly important to break these apart into strict units because they're just not important enough.